Chap_API_Server.tex

```diff
@@ -1218,12 +1226,14 @@ \subsection{\code{PMIx_server_setup_application}}
 \argin{cbdata}{Data to be passed to the \refarg{cbfunc} callback function (memory reference)}
 \end{arglist}
 
-Returns one of the following:
+% TODO: Does this allow for PMIX_OPERATION_SUCCEEDED (I added it)
```

Comment:
I'd keep it - per above.

Reply:
Again, this function cannot return "operation succeeded" - it must call the cbfunc, as per above.